contents/uix/uix.md

```diff
@@ -0,0 +1,47 @@
+# UI/UX
+
+Author: [Ang Shi Ya](https://github.com/AngShiYa)
+
+## Overview
+
+User Interface (UI) and User Experience (UX) are terms often used interchangeably. In fact, these two terms are quickly becoming dangerous buzzwords that are used inaccurately, creating massive confusion in the industry. A simple google search will show that UI Designer and UX Designer jobs are listed under similar titles and descriptions. 
+
+In addition to helping us identify and apply for the right role, understanding the key differences between UI and UX helps us in using the right concept in our own applications. This document will address the difference between UI and UX as well as some of the techniques used in UI Design and UX Design.
+
+## User Interface vs User Experience
+
+UI is about the design of the buttons, the layout of the website and the responsiveness of the application. It encompasses the presentation and interactivity of the application. On the other hand, UX is concern with the ease of use and the pleasure provided through using the application. In short, UI is about the look and feel while UX is about customer satisfaction.
+
+Due to the overlapping nature of UI and UX, you might still have some questions such as *'Doesn't improving the UI also improve the UX?'* Indeed, it does. In fact, UI **can** be part of UX.
+
+> Note that I used **can** because UI is no longer a necessity to UX due to the emergence of [zero UI](http://blog.careerfoundry.com/ui-design/what-is-zero-ui), which won't be covered here.
+
+To gain a better understanding, let's look at real life examples of good UI but bad UX. A prime example would be the [Mystery Meat Navigation](https://www.techinasia.com/talk/material-design-mystery-meat-navigation-problem) problem. It refers to buttons and links that do not explain themselves; users have to click on them to find out where it leads. In essence, the pursuit of clean and neat UI has resulted in omission of details that provide clarity for the user. This results in bad UX because people do not like to puzzle over how things work. A good UX should never make the user think.
+
+If you would like to read more about the differences between UI and UX, here are a few articles:
+
+* [The Difference Between UX and UI Design-A Layman’s Guide](http://blog.careerfoundry.com/ui-design/the-difference-between-ux-and-ui-design-a-laymans-guide/) - provides breakdown of the disciplines involved.
+* [What’s the difference between UI and UX?](https://www.usertesting.com/blog/2016/04/27/ui-vs-ux/) - provides opinions from UX experts.
+* [The superpowers of UX & UI designers](https://www.ymedialabs.com/ux-vs-ui/) - provides illustrations to highlight the differences.
+
+## UI Design
+
+UI Design is highly dependent on your sense of visual design, but thanks to the rise of flat and minimalist design you don't have to be a photoshop expert to get started on UI design. 
```

Comment:
Not sure this sentence makes logical sense. Why does the rise of flat design removes the need to know photoshop or the dependency of sense of visual design?